src/Prestige.ts

```diff
@@ -195,6 +199,9 @@ export function prestigeSourceFile(isFlume: boolean): void {
   // We must kill all scripts before doing anything else.
   prestigeWorkerScripts();
 
+  // Close coding contract modal
+  CodingContractEventEmitter.emit({ type: "close" });
+
```

Comment:
Someday, we ought to refactor these so they can have more of a shared impl. It's a mess.